package.json

```diff
@@ -26,61 +28,66 @@
   },
   "homepage": "https://github.com/Microsoft/PowerBI-visuals-tools#readme",
   "dependencies": {
+    "@typescript-eslint/parser": "^5.60.1",
     "assert": "^2.0.0",
     "async": "^3.2.4",
     "browserify-zlib": "^0.2.0",
     "buffer": "^6.0.3",
-    "chalk": "3.0.0",
-    "commander": "4.1.1",
-    "compare-versions": "^3.6.0",
+    "chalk": "^5.3.0",
+    "commander": "^11.0.0",
+    "compare-versions": "^5.0.3",
     "console-browserify": "^1.2.0",
```

Comment:
Most of the following libraries are not used. Please remove it if it's necessary:

- console-browserify
- constants-browserify
- crypto-browserify
- domain-browser
- https-browserify
- os-browserify
- path-browserify
- stream-browserify
- times-browserify

and ets

Reply:
Decided to bring them back in webpack config. Since webpack 5 it's [required](https://webpack.js.org/configuration/resolve/#resolvefallback)